proto/authzed/api/v1/00_additional_validation.pb.go

```diff
@@ -0,0 +1,102 @@
+// The contents of this file are hand-written to add AdditionalValidation to select message types
```

Comment:
Don't name this pb.go -- that's explicitly used as the naming convention for marking files as generated

Reply:
Well, we use it for the other non-generated files, but I can drop it if you like